Config/Routes.js

```diff
@@ -1,10 +1,20 @@
-const ProductController = require("../Controllers/ProductController");
+const DbService = require('../Services/DbService')
+const ProductController= require("../Controllers/ProductController");
 
 function routes(app) {
     app.get('/products', ProductController.getAllProducts)
     app.post('/products', ProductController.addSingleProduct)
+    app.put('/products', ProductController.updateSingleProduct)
+    app.put('/products', ProductController.invalidRoute)
```

Comment:
should we set up delete pathways as invalid routes as well?

Reply:
Good shout, have done